modin/core/execution/dask/implementations/pandas_on_dask/io/io.py

```diff
@@ -31,6 +31,9 @@
     SQLDispatcher,
 )
 from modin.core.storage_formats.pandas.parsers import (
+    ExperimentalCustomTextParser,
+    ExperimentalPandasPickleParser,
```

Comment:
Since there is now one factory for both modes, importing experimental functionality in a regular module will be a common thing.

Reply:
I mean should we move experimental parsers to modin.experimental.core.storage_formats.pandas.parsers?

Reply:
Got it. I think with the current implementation it would be more consistent to move the code. However, the following question arises: why do we divide the code into experimental and non-experimental, only for IO functions? Should we do the same for all other cases of experimental code (groupby, asynchronous read, numpy module implementation and maybe some other cases)?

Reply:
Will do separately.